include/eve/detail/function/simd/arm/sve/sum.hpp

```diff
@@ -0,0 +1,54 @@
+//==================================================================================================
+/*
+  EVE - Expressive Vector Engine
+  Copyright : EVE Project Contributors
+  SPDX-License-Identifier: BSL-1.0
+*/
+//==================================================================================================
+#pragma once
+
+#include <eve/concept/value.hpp>
+#include <eve/conditional.hpp>
+#include <eve/detail/implementation.hpp>
+#include <eve/detail/remove_garbage.hpp>
+
+// remove_garbage
+namespace eve::detail
+{
+template<conditional_expr C, typename T, typename N>
+EVE_FORCEINLINE auto
+sum_(EVE_SUPPORTS(sve_), C const& cond, wide<T, N> v) noexcept -> T
+requires sve_abi<abi_t<T, N>>
+{
+  // Fix mask to not touch garbage lanes if it is not an ignore_*
+  auto m = cond.mask(as(v));
+  if constexpr(!relative_conditional_expr<C>) m = remove_garbage(cond.mask(as(v)));
```

Comment:
`remove_garbage(m)` - don't call mask twice.

We need a function for this.. But maybe it's not that important though.

Reply:
remove_garbage works only on wide and logical. If m is a conditional, I need to call remove_garbage on its logical mask.